.github/workflows/benchmarks.yml

```diff
@@ -0,0 +1,78 @@
+name: Benchmarking
+on:
+  push:
+    branches:
+      - master
```

Comment:
Does this need to run when we merge-in PR's to master? In that case, aren't the two cases of `lightning.qubit` the same, i.e., it's comparing `master` to itself?

Reply:
No, thanks for the catch!